docs/data/material/customization/palette/palette.md

```diff
@@ -38,173 +58,144 @@ If any of the:
+To add custom colors either provide the tokens manually or generate them using the `augmentColor` utility:
 
-### Adding new colors
+### Provide tokens manually
 
-You can add new colors inside and outside the palette of the theme as follows:
+The most straightforward approach is to define all tokens—`main`, `light`, `dark`, and `contrastText`—manually:
 
-```js
-import { createTheme } from '@mui/material/styles';
+{{"demo": "ManuallyProvideCustomColor.js" }}
 
-const theme = createTheme({
-  status: {
-    danger: '#e53e3e',
-  },
-  palette: {
-    primary: {
-      main: '#0971f1',
-      darker: '#053e85',
-    },
-    neutral: {
-      main: '#64748B',
-      contrastText: '#fff',
-    },
-  },
-});
+If you need to manipulate colors, then `@mui/material/styles` provides [a set of utilities](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/styles/index.d.ts#L52-L67) to help with this. The following example uses the `alpha` and `getContrastRatio` utils to define tokens using opacity:
+
+{{"demo": "UsingStylesUtils.js" }}
+
+### Generate tokens using augmentColor utility
+
+Alternatively, you can generate the `light`, `dark` and `contrastText` tokens using the palette's `augmentColor` utility,
+which is the same function used for the default palette colors.
+This requires creating the theme in two steps and providing the `main` token on which the other will be based on:
+
+{{"demo": "UsingAugmentColor.js" }}
+
+The [contrast threshold](#contrast-threshold) and [tonal offset](#tonal-offset) values
+will apply for the colors defined using this utility.
+
+### Using in components
+
+After adding a custom color,
+you will be able to use it in components the same way default palette colors are used:
+
+```js
+<Button color="custom">
 ```
 
-### TypeScript
+### Typescript
 
-You have to use [module augmentation](/material-ui/guides/typescript/#customization-of-theme) to add new variables to the `Palette` and `PaletteOptions`.
+If you're using Typescript, then you need to use [module augmentation](/material-ui/guides/typescript/#customization-of-theme) for custom colors.
+
+To add a custom color to the palette,
+you'll have to add it to the `Palette` and `PaletteOptions` interfaces:
 
 <!-- tested with packages/mui-material/test/typescript/augmentation/paletteColors.spec.ts -->
 
 ```ts
 declare module '@mui/material/styles' {
-  interface Theme {
-    status: {
-      danger: React.CSSProperties['color'];
-    };
+  interface Palette {
+    custom: Palette['primary'];
   }
 
-  interface ThemeOptions {
-    status: {
-      danger: React.CSSProperties['color'];
-    };
+  interface PaletteOptions {
+    custom?: PaletteOptions['primary'];
   }
+}
+```
 
-  interface Palette {
-    neutral: Palette['primary'];
-  }
+To use the custom color for the `color` prop of a component,
+you'll need to add it to the component's `PropsColorOverrides` interface.
+For example if you need to use it on the `Button` component add it like this:
```

Comment:
Copy editing; no back ticks around component names (this is an outdated convention that we're trying to move past). 

```suggestion
The example below shows how to do this with a Button component:
```